dev_guide/builds.adoc

```diff
@@ -1557,3 +1557,114 @@ Docker and Source builds set the following labels on output images:
 |*io.openshift.build.source-location*
 |Source URL for the build
 |===
+
+
+[[using-external-artifacts]]
+= Using External Artifacts During The Build
+
+In order to use external artifacts during the build you don't need any special
+support from OpenShift platform. You could download and use any artifacts
+as you want.
```

Comment:
I mean that there is no special commands, settings and so on. How I could improve this?